homeassistant/components/vera/common.py

```diff
@@ -0,0 +1,122 @@
+"""Common vera code."""
```

Comment:
This whole module, common.py, only seems necessary to support multiple controllers.

Please break out this change to a separate PR. It's ok to do that PR first and do this config entry PR later.

Reply:
All the functions and data structures in common are used to support the creation of entities from a platform's async_setup_entry function.

Reply:
Yes, they are used but they don't seem necessary. The point seems to be to prepare for multiple controllers. We should keep each PR minimal.

Reply:
Further cleanup complete.